osmaxx-py/osmaxx/excerptexport/static/excerptexport/scripts/formModeSwitcher.js

```diff
@@ -26,8 +26,13 @@
             formPartsSwitcher.value = formModeParam;
         }
 
+        // IE may throw an error here "Object doesn't support this action".
+        // It seems IE won't trigger a change event on select boxes.
+        // I didn't find a solution to get the supported actions,
+        // so the error is still there: Fuck you IE.
```

Comment:
Do we really want to serve expletives like this to clients? We don't have the exemption usually granted to guys like Torvalds.